applications/nrf_desktop/README.rst

```diff
@@ -282,35 +296,40 @@ Configuration files are provided for different build types for each supported bo
     `Selecting a build type`_ is optional.
     The ``ZDebug`` build type is used by default in nRF Desktop if no build type is explicitly selected.
 
-The following build types are available in nRF Desktop:
+The following build types are available for various boards in nRF Desktop:
 
 * ``ZRelease`` -- Release version of the application with no debugging features.
+* ``ZReleaseB0`` -- ``ZRelease`` build type with the support for the B0 bootloader enabled (for :ref:`background DFU <nrf_desktop_bootloader_background_dfu>`).
+* ``ZReleaseMCUBoot`` -- ``ZRelease`` build type with the support for the MCUBoot bootloader enabled (for :ref:`serial recovery DFU <nrf_desktop_bootloader_serial_dfu>`).
 * ``ZDebug`` -- Debug version of the application; the same as the ``ZRelease`` build type, but with debug options enabled.
-* ``ZDebugWithShell`` -- ``ZDebug`` build type with the shell enabled.
-* ``ZReleaseB0`` -- ``ZRelease`` build type with the support for the bootloader enabled.
 * ``ZDebugB0`` -- ``ZDebug`` build type with the support for the bootloader enabled.
```

Comment:
I would mention that this is B0 bootloader (to be consistent with realease configurations description)

Reply:
Fixed.